packages/web-api/src/WebClient.spec.js

```diff
@@ -1148,132 +1148,79 @@ describe('WebClient', function () {
         });
   });
 
-  describe('has all admin.inviteRequests.* APIs', function () {
-    function verify(runApiCall, methodName, expectedBody, done) {
-      const scope = nock('https://slack.com')
-        .post(`/api/${methodName}`)
+  describe('admin.inviteRequests.* APIs', function () {
+    const client = new WebClient(token);
+
+    beforeEach(function() {
+      this.scope = nock('https://slack.com')
+        .post(/api/)
         .reply(200, function (_uri, body) {
           return { ok: true, body: body };
         });
-      runApiCall
-        .then((res) => {
-          assert.equal(res.body, expectedBody);
-        })
-        .catch((err) => {
-          assert.fail(err);
-        })
-        .finally(() => {
-          scope.done();
-          done();
-        });
-    }
-    const client = new WebClient(token);
+    });
 
-    it('can call admin.inviteRequests.approve', function (done) {
-      verify(
-        client.admin.inviteRequests.approve({ team_id: 'T123', invite_request_id: 'I123' }),
-        'admin.inviteRequests.approve',
-        'team_id=T123&invite_request_id=I123',
-        done,
-      );
+    it('properly serializes arguments to admin.inviteRequests.approve', async function () {
```

Comment:
I took the liberty of renaming the tests here as well to try to make them more representative of what we are actually testing here. Given we are controlling the response of the API to (via `nock`) to simply return the request body, in effect these tests only check that:

1. The relevant methods on the client exist, can be called and return a response, and 
2. That depending on the argument type, the arguments are serialized properly.

As a result I decided to rename these tests to be "check that arguments are serialized properly" since that is in effect what we are exercising in these tests.

Reply:
IMO this renaming makes it way more clear what is happening in these tests - thanks 👍 

For context (and later possible eyes on this thread), axios/axios#4036 is the related issue with a succinct summary.